Retail.lua

```diff
@@ -486,33 +541,42 @@ addon.Spells = {
     [213871] = { type = BUFF_DEFENSIVE }, -- Bodyguard
     [227847] = { type = IMMUNITY }, -- Bladestorm (Arms)
         [46924] = { type = IMMUNITY, parent = 227847 }, -- Bladestorm (Fury)
-        [152277] = { type = IMMUNITY, parent = 227847 }, -- Ravager
-    [228920] = { type = BUFF_DEFENSIVE }, -- Ravager
     [236077] = { type = CROWD_CONTROL }, -- Disarm
         [236236] = { type = CROWD_CONTROL, parent = 236077 }, -- Disarm
-    [236273] = { type = BUFF_DEFENSIVE }, -- Duel
+    [236273] = { type = CROWD_CONTROL }, -- Duel
```

Comment:
```suggestion
    [236273] = { type = BUFF_DEFENSIVE }, -- Duel
```
I'm not sure I agree with this being put CC, as it doesn't prevent the player from doing anything (maybe should be `DEBUFF_OFFENSIVE` ? I can't remember if this is actually a buff on the warrior or debuff on the target)

Reply:
It's definitely a debuff on target. I put it into CC because I believe people use it like a CC spell to peel for others, i.e. you'd use this on an enemy DPS while he's trying to kill your teammate to stop his damage.

DEBUFF_OFFENSIVE would make sense too though.